modules/ROOT/pages/batch-error-handling-faq.adoc

```diff
@@ -1,17 +1,15 @@
-= Handling Errors During Batch Job
+= Handling Errors During Batch Jobs
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
 == Overview
 
-Handling errors inside a batch job can be complicated considering the huge amount of data this scope is designed to take. +
-When processing 1 million records, it's almost impossible to log everything. Logs would become huge, unreadable, and would take a massive toll on the performance of your processing. +
-On the other hand, logging too little would make it impossible for you to know what went wrong. And considering that if out of those 1 million records, 30 thousand of them failed, not knowing what’s wrong with them can turn your debugging task very problematic. This trade-off is not simple to overcome.
+Handling errors inside a batch job is complicated by the enormous amount of data this scope is designed to handle. For example, logs from processing 1 million records become huge and unreadable, and the processing required for the logs causes major performance issues. However, logs are necessary to understand issues that occur. Without logs, debugging the failure of 30,000 records is not practical. 
 
 == Logs of Failing Records Inside a Batch Step
 
-When processing a batch job instance, a processor inside a batch step may fail or raise an error. When this occurs – perhaps because of corrupted or incomplete record data – Mule logs the stack traces following this logic:
+When processing a batch job instance, a processor inside a batch step may fail or raise an error. When this occurs, for example, because of corrupted or incomplete record data, Mule logs the stack traces following this logic:
```

Comment:
```suggestion
When processing a batch job instance, a processor inside a batch step may fail or raise an error, for example, because of corrupted or incomplete record data. When such an issue occurs, Mule logs the stack traces following this logic:
```